protos/Crane.proto

```
@@ -19,21 +19,21 @@
 syntax = "proto3";
 
 package crane.grpc;
-option go_package = "/protos";
+option go_package = "CraneFrontEnd/generated/protos";
 
 import "google/protobuf/empty.proto";
 import "google/protobuf/timestamp.proto";
 import "PublicDefs.proto";
```

Buf (1.54.0)

import "PublicDefs.proto": file does not exist

(COMPILE)